src/components/POLine/POLine.js

```diff
@@ -164,7 +164,7 @@ class POLine extends Component {
             <br />
           </Accordion>
           <Accordion label="Vendor" id="Vendor">
-            <VendorView initialValues={initialValues} {...this.props} />
+            <VendorView vendor={initialValues.vendor_detail} {...this.props} />
```

Comment:
basically, it's not a `vendor` itself, it's `vendorDetails`... Maybe it's better to change